tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+envlist=
+    py27,
+    py34,
+    py35
+
+[flake8]
+max-line-length= 100
+exclude= tests/*
+
+[testenv]
+commands=py.test --tb native {posargs:tests}
```

Comment:
Actually, you're right.  I was actually in the wrong window and running tests on a different repo :)

Fixing for real this time.